tests/metrics_test.py

```diff
@@ -830,6 +830,7 @@ def test_store_race(self):
                             pipeline="from-sources", user_tags={"os": "Linux"}, track=t, track_params={"shard-count": 3},
                             challenge=t.default_challenge, car="defaults", car_params={"heap_size": "512mb"}, plugin_params=None,
                             total_laps=12,
+                            track_revision="abc1",
```

Comment:
We should also support and test `team_revision`, right?

Reply:
As mentioned in the [above comment](https://github.com/elastic/rally/pull/725#discussion_r310100420) I think we need to test for `team_revision` to ensure things work as they are supposed?